### PR TITLE
DOMA-4422 change platform description texts

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

**Review by pahaz — APPROVED** [review]

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/lang/en/en.json [hunk 1]
```diff
@@ -128,9 +128,9 @@
   "filters.Apply": "Apply",
   "filters.NewFilter": "New filter",
   "filters.Template": "Template",
-  "filters.NewTemplateLabel": "Template name (Please fill it only if you want to save the template. To apply a filter leave this field empty.)",
+  "filters.NewTemplateLabel": "Fill this field if you want to save this template for the future usage",
   "filters.TemplateLabel": "Template name",
-  "filters.NewTemplatePlaceholder": "For example, \"my filters set\"",
+  "filters.NewTemplatePlaceholder": "For example, my filters set",
```
Comment (akarjakin) [inline apps/condo/lang/en/en.json:133]: ex.: My filter set

File: apps/condo/lang/en/en.json [hunk 2]
```diff
@@ -803,15 +803,15 @@
...
+  "employee.role.Administrator.description": "The administrator has full access to the data and functions of the Houses. He can delete, create or edit anything he wants. Administrators control the created organizations. An organization can have an unlimited number of administrators.",
   "employee.role.Dispatcher.name": "Dispatcher",
-  "employee.role.Dispatcher.description": "Dispatchers interact with the client and the internal services of the CM. Dispatchers fill out applications, leave comments, change statuses, enter information on the readings of metering devices and issue keys. Dispatchers have access to information about customers, charges and technical personnel, but cannot edit, add or delete anything.",
+  "employee.role.Dispatcher.description": "Dispatcher interact with the client and the internal services of the CM. Dispatchers fill out applications, leave comments, change statuses, enter information based on meter readings and issue keys. Dispatchers have access to information about customers, accruals and technical staff, but cannot edit, add or delete anything.",
```
Comment (akarjakin) [inline apps/condo/lang/en/en.json:808]: Dispatcher interacts with...

File: apps/condo/lang/en/en.json [hunk 3]
```diff
@@ -803,15 +803,15 @@
   "employee.emptyList.title": "You are blocked",
   "employee.emptyList.description": "Your employer has blocked you from the organization",
   "employee.role.Administrator.name": "Administrator",
-  "employee.role.Administrator.description": "The administrator has full access to the data and functions of the Houses: he can delete, create or edit anything he wants. Administrators control the created organizations. An organization can have an unlimited number of administrators.",
+  "employee.role.Administrator.description": "The administrator has full access to the data and functions of the Houses. He can delete, create or edit anything he wants. Administrators control the created organizations. An organization can have an unlimited number of administrators.",
```
Comment (akarjakin) [inline apps/condo/lang/en/en.json:806]: He/she

File: apps/condo/lang/en/en.json [hunk 4]
```diff
@@ -803,15 +803,15 @@
...
+  "employee.role.Dispatcher.description": "Dispatcher interact with the client and the internal services of the CM. Dispatchers fill out applications, leave comments, change statuses, enter information based on meter readings and issue keys. Dispatchers have access to information about customers, accruals and technical staff, but cannot edit, add or delete anything.",
   "employee.role.Manager.name": "Manager",
-  "employee.role.Manager.description": "The manager is the manager of the house: he interacts with clients, monitors the quality, deadlines and fulfillment of orders, controls the finances of the house - manages a house or a group of houses. Within the designated homes, the manager has access to ticket management, employee management, household reports, charges and payments.",
+  "employee.role.Manager.description": "The manager is the manager of the house. He interacts with clients, monitors the quality, deadlines and fulfillment of orders, controls the finances of the house - manages a house or a group of houses. Within the designated homes, the manager has access to ticket management, employee management, household reports, charges and payments.",
```
Comment (akarjakin) [inline apps/condo/lang/en/en.json:810]: He/she

File: apps/condo/lang/en/en.json [hunk 5]
```diff
@@ -803,15 +803,15 @@
...
   "employee.role.Foreman.name": "Foreman",
-  "employee.role.Foreman.description": "The site foreman is responsible for the performance and quality of work on the site. The site master distributes tasks to performers and configures routing.",
+  "employee.role.Foreman.description": "The foreman is responsible for the performance and quality of work on the site. The site master distributes tasks to performers and configures routing.",
   "employee.role.Technician.name": "Technician",
-  "employee.role.Technician.description": "The technician works with tasks and performs work. The technician does not have access to other sections and capabilities of the system, except for applications.",
+  "employee.role.Technician.description": "The technician works with client tickets. He does not have access to other sections and capabilities of the system.",
```
Comment (akarjakin) [inline apps/condo/lang/en/en.json:814]: He/she

**Pull request merged**